test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/BasicTagHelperWithMixedQuotesAndNonQuotes.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@tagHelperPrefix THS
```

Comment:
Whaaa, nononononono. I was saying that you could just intermingle quotes/non-quotes in existing tests that have design-time tests for them already. Same with below. These things don't deserve their own dedicated tests